src/gowrapper/ckks/params.go

```diff
@@ -63,6 +64,37 @@ func lattigo_getDefaultPQParams(paramEnum uint8) Handle6 {
 	return marshal.CrossLangObjMap.Add(unsafe.Pointer(&params))
 }
 
+//export lattigo_newParameters
+func lattigo_newParameters(logN uint64, qi *C.constULong, numQi uint8, pi *C.constULong, numPi uint8, logScale uint8) Handle6 {
+	size := unsafe.Sizeof(uint64(0))
+
+	Qi := make([]uint64, numQi)
+	qiPtr := uintptr(unsafe.Pointer(qi))
+	for i := range Qi {
+		Qi[i] = *(*uint64)(unsafe.Pointer(qiPtr + size*uintptr(i)))
```

Comment:
This doesn't make sense. Why are you multiplying `size` (which is probably a `size_t`) by a pointer? Multiplying by a pointer (in general) doesn't make sense. While this might work, does this also work (and make more sense)?

`qiPtr + size * i` or `qiPtr + uintptr(size * i)`

Reply:
See https://stackoverflow.com/a/32701024/925978. Go does not appreciate me mucking about with memory, but this seems to be the way to do it.